x-pack/test/functional/apps/canvas/reports.ts

```diff
@@ -192,14 +192,14 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           endobj
           21 0 obj
           <<
-          /Length 250
+          /Length 254
```

Comment:
For review from shared UX, I noticed these tests were removed on main.  Have we run into this type of change before?